UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/IFileUtility.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UKHO.PeriodicOutputService.Common.Utility
```

Comment:
Namespace should be UKHO.PeriodicOutputService.Common.Utilities to correspons to the file location.

Reply:
Renamed